fission/vite.config.ts

```diff
@@ -1,76 +1,85 @@
-import { defineConfig } from 'vitest/config'
-import * as path from 'path'
-import react from '@vitejs/plugin-react-swc'
-import glsl from 'vite-plugin-glsl';
+import { defineConfig } from "vitest/config"
+import * as path from "path"
+import react from "@vitejs/plugin-react-swc"
+import glsl from "vite-plugin-glsl"
 
-const basePath = '/fission/'
+const basePath = "/fission/"
 const serverPort = 3000
 const dockerServerPort = 80
 
-const useLocal = false
+const useLocal = true
 
 // https://vitejs.dev/config/
 export default defineConfig({
-    plugins: [react(), /* viteSingleFile() */ glsl({
-    include: [                   // Glob pattern, or array of glob patterns to import
-      '**/*.glsl', '**/*.wgsl',
-      '**/*.vert', '**/*.frag',
-      '**/*.vs', '**/*.fs'
+    plugins: [
+        react(),
+        /* viteSingleFile() */ glsl({
```

Comment:
I see that the formatter caught this file. (Which is totally awesome!) But we should probably remove the `viteSingleFile()` plugin comment as a part of this formatting change.

Reply:
yes! that's so cool!